### ISSUE TITLE
Buzon de entrega

### ISSUE THREAD

**Issue by bgsamilla** [IIC2523/2020-2-Tarea1#18]
Hola!

Cuando estará disponible en buzon de entrega para la tarea?

Se podrán hacer entregas parciales?

**Comment by FranLucchini** [comment]
La verdad es que no lo había pensado. Voy a preguntarle al profe y mando un aviso.

**Comment by FranLucchini** [comment]
@bgsamilla ya está abierto el buzón de entrega y le puse que tenga envíos ilimitados, entonces se puede subir la tarea cuantas veces quieren (se pueden hacer entregas parciales :wink:)

**Comment by bgsamilla** [comment]
Que nombre de archivo debemos usar?

El enunciado dice que debe ser **NúmeroDeAlumno_IIC2523-2_Tarea1**
Mientras que el buzon dice **NúmeroDeAlumnoIIC2523-2Tarea1**

Es un detalle pero si van a automatizar el proceso de correccion puede dar problemas.

**Comment by FranLucchini** [comment]
Cierto, cierto, toda la razón. Sigue la del enunciado, voy a cambiar la del buzón que la copié mal.
